com.ibm.wala.core.tests/src/com/ibm/wala/core/tests/callGraph/LambdaTest.java

```diff
@@ -135,4 +139,95 @@ protected void checkCompareToCalls(CallGraph cg, Set<CGNode> ctnodes, String x,
     Assert.assertEquals("expected one call to compareTo", expected, count);
     System.err.println("found " + count + " compareTo calls in " + sfnode);
   }
+
+  @Test
+  public void testMethodRefs()
+      throws ClassHierarchyException, IllegalArgumentException, CancelException, IOException {
+
+    AnalysisScope scope =
+        CallGraphTestUtil.makeJ2SEAnalysisScope(
+            TestConstants.WALA_TESTDATA, CallGraphTestUtil.REGRESSION_EXCLUSIONS);
+    ClassHierarchy cha = ClassHierarchyFactory.make(scope);
+    Iterable<Entrypoint> entrypoints =
+        com.ibm.wala.ipa.callgraph.impl.Util.makeMainEntrypoints(scope, cha, "Llambda/MethodRefs");
+    AnalysisOptions options = CallGraphTestUtil.makeAnalysisOptions(scope, entrypoints);
+
+    CallGraph cg =
+        CallGraphTestUtil.buildZeroCFA(options, new AnalysisCacheImpl(), cha, scope, false);
+
+    Function<String, MethodReference> getTargetRef =
+        (klass) -> {
```

Comment:
Use expression lambda here instead of statement lambda:

``` java
        (klass) -> MethodReference.findOrCreate(
            TypeReference.findOrCreate(
                ClassLoaderReference.Application, "Llambda/MethodRefs$" + klass),
            Atom.findOrCreateUnicodeAtom("target"),
            Descriptor.findOrCreateUTF8("()V"));
```